lib/fastlane/plugin/firebase_test_lab/helper/ios_validator.rb

```diff
@@ -13,10 +13,10 @@ def self.validate_ios_app(file_path)
               UI.user_error!("app verification failed: There must be only one .xctestrun files in the ZIP file.")
             end
 
-            conf = Plist.parse_xml(xctestrun_files.first.get_input_stream)
-            unless conf.size == 1
-              UI.user_error!("The app bundle may contain only one scheme, #{conf.size} found")
-            end
+            # conf = Plist.parse_xml(xctestrun_files.first.get_input_stream)
```

Comment:
You should really delete that code - the history is contained in the git repo, so if we ever need it back it will be there.